### PR TITLE
fix: #31 and #32 Content Security Policy warning and refuses to load Google fonts. and content security policy for images

### PR DESCRIPTION
**Pull request by parthnamdev**
fixed issue #31 

fixed issue #32 

you may review it. hope this helps

### CODE REVIEW

**Review by acrosman — CHANGES_REQUESTED** [review]
I'm not going to merge in the use of unsafe-inline.  Please use a different solution.

**Comment by parthnamdev** [comment]
i have tried using sha256 hash. could be please check if this works for you.

**Comment by acrosman** [comment]
Thank you for working on this.  I'm going to go ahead and mark it hacktoberfest-accepted so you get credit there (in case I forget to check in on it again).  

Generally, since the project has a style sheet and an existing theme it would be better to use classes from boot strap or put the style into the stylesheet and not have it defined inline in the HTML. That would remove the need for the SHA and keep the various details more properly separated.

File: app/console.html [hunk 1]
```diff
@@ -2,16 +2,21 @@
 <html>
     <head>
         <meta charset="UTF-8">
-        <meta http-equiv="Content-Security-Policy" content="default-src 'self'">
+        <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self' https://fonts.googleapis.com 'sha256-osjHfNDP4at4lg2rARZvLE6bzOq48Q1BhccVPpPKt1Q='; font-src 'self' https://fonts.gstatic.com">
         <link rel="stylesheet" href="../node_modules/bootswatch/dist/cyborg/bootstrap.min.css">
         <link rel="stylesheet" href="../node_modules/jquery.json-viewer/json-viewer/jquery.json-viewer.css">
         <link rel="stylesheet" href="console.css">
         <title>ElectronForce Console</title>
+        <style>
```
Comment (acrosman) [inline app/console.html:10]: This style should be moved to the style sheet at app/dashboard.css or a class from bootstrap should be used in it place.

Comment (parthnamdev) [reply app/console.html]: yes, moving style to the stylesheet is better. I moved it as you suggested. Actually I am new to open source and learning new skills. 

And thank you for adding the hacktoberfest-accepted label.

**Pull request merged**